Header header
float64 timestamp
float64 smooth_x
float64 smooth_y
float64 smooth_z
float64 latitude
float64 longitude
float64 altitude
float32 vel_north
float32 vel_east
float32 vel_up
float32 speed
float32 track
float64 roll
float64 pitch
float64 yaw
float64 rate_roll
float64 rate_pitch
float64 rate_yaw
float64 accel_x
float64 accel_y
float64 accel_z
float64 wander
uint32  id
int32   postprocess_code
float64 hardware_timestamp
int32   hardware_time_mode
